app/Program.cs

```diff
@@ -1,19 +1,30 @@
-﻿using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Hosting;
+﻿using app;
+using app.Data;
+using JsonApiDotNetCore.Configuration;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
 
-namespace app
+var builder = WebApplication.CreateBuilder(args);
+
+string connectionString = builder.Configuration["Data:DefaultConnection"];
+builder.Services.AddDbContext<AppDbContext>(options => options.UseNpgsql(connectionString));
+
+builder.Services.AddJsonApi<AppDbContext>(options =>
 {
-    public class Program
-    {
-        public static void Main(string[] args)
-        {
-            CreateHostBuilder(args).Build().Run();
-        }
+    options.UseRelativeLinks = true;
+    options.IncludeTotalResourceCount = true;

```

Comment:
I think this should also contain:
```c#
options.SerializerSettings.Converters.Add(new StringEnumConverter());
```
so that `TodoItem.Priority` gets serialized like to used to.